internal/catalogmetadata/filter/bundle_predicates.go

```diff
@@ -60,6 +60,22 @@ func WithBundleImage(bundleImage string) Predicate[catalogmetadata.Bundle] {
 	}
 }
 
+func WithBundleName(bundleName string) Predicate[catalogmetadata.Bundle] {
+	return func(bundle *catalogmetadata.Bundle) bool {
+		return bundle.Name == bundleName
+	}
+}
+
+func WithBundleVersion(bundleVersion string) Predicate[catalogmetadata.Bundle] {
```

Comment:
This isn't needed at all, I'll remove it and use an existing in semver range predicate that checks for version equality.